site/components/footer/index.js

```diff
@@ -20,31 +21,43 @@ import mapSVG from './SVG/map.svg';
 
 const cx = classnames.bind(styles);
 
+const trackAnalytics = title => () => ReactGA.event({
+  category: 'FooterNavigation',
+  action: title,
+  label: 'click',
+});
```

Comment:
This works on Cmd+Click and should be the same for Ctrl+click (for windows), but doesn't work with middle click or right click context menu. Will look into a possible fix. Might be better just to put this out and set up a bug for this so we don't forget about it?